src/Core/Commander/Interfaces/ApiInterface/ApiGlobe.js

```diff
@@ -13,11 +14,11 @@ import TileProvider from 'Core/Commander/Providers/TileProvider';
 import CoordCarto from 'Core/Geographic/CoordCarto';
 import Ellipsoid from 'Core/Math/Ellipsoid';
 import Projection from 'Core/Geographic/Projection';
+import 'Core/Commander/Interfaces/Event';
```

Comment:
I'd rather add the polyfills to the WebPack entry-point, like we did for the others (fetch and promises)